.gitlab-ci.yml

```diff
@@ -165,15 +165,15 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v32368242-2e3b6ef4
+  DATADOG_AGENT_BUILDIMAGES: v32377621-ce42cb8e
   DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v32368242-2e3b6ef4
+  DATADOG_AGENT_WINBUILDIMAGES: v32377621-ce42cb8e
   DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v32368242-2e3b6ef4
+  DATADOG_AGENT_ARMBUILDIMAGES: v32377621-ce42cb8e
   DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32368242-2e3b6ef4
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32377621-ce42cb8e
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v32368242-2e3b6ef4
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v32377621-ce42cb8e
   DATADOG_AGENT_BUILDERS: v28719426-b6a4fd9
```

Comment:
If I understand correctly what is the purpose of your change I would have expected you to add a commit that would replace the `builders` version by the buildimage one, wdyt?